src/memote/network.jl

```diff
@@ -0,0 +1,184 @@
+#=
+This file contains a collection of tests based on Memote. See Lieven, C., Beber,
+M.E., Olivier, B.G. et al. MEMOTE for standardized genome-scale metabolic model
+testing. Nat Biotechnol 38, 272–276 (2020).
+https://doi.org/10.1038/s41587-020-0446-y for details.
+=#
+
+"""
+$(TYPEDSIGNATURES)
+
+Return the ratio of the absolute maximum and minimum value of the nonzero
+coefficients in the stoichiometric matrix of `model`.
+"""
+stoichiometric_max_min_ratio(model) =
+    /(reverse(extrema(abs, x for x in stoichiometry(model) if x != 0.0))...)
```

Comment:
do not invert™. Maybe just call it stoichiometric_magnitude_extrema and return min/max so that everyone can mess it up for themselves :D

Reply:
of this doesn't invert, only takes a ratio of two real numbers :)